libparsec/crates/crypto/src/common/sequester.rs

```diff
@@ -21,9 +21,11 @@ pub(crate) fn serialize_with_armor(
     // It is important `output` has a constant size given this
     // is how it is retrieved during decryption
     assert!(output.len() <= key_size_bytes);
-    let mut res = vec![0; algo.len() + 1 + key_size_bytes + data.len()];
+    let mut res = vec![0; algo.len() + 1 + 2 + 1 + key_size_bytes + data.len()];
```

Comment:
Can you use constants for those numeric values ?

Reply:
Done